blockchain/chain.go

```diff
@@ -1309,6 +1412,7 @@ func New(config *Config) (*BlockChain, error) {
 		blocksPerRetarget:   int32(targetTimespan / targetTimePerBlock),
 		index:               newBlockIndex(config.DB, params),
 		hashCache:           config.HashCache,
+		bestChain:           newChainView(nil),
```

Comment:
Is it maybe worth having a chainView constructor that accepts a capacity since that can be looked up from the DB at startup?